### PR TITLE
Signed in customer access to create account page

### LINKED ISSUE #1549: [bug]: Sign in customer access create account page
**Describe the bug**
Sign in customer is able to access create account page.

**To Reproduce**
1.  Sign in with customer detail .
2.  Now open create account page
3.  You are able to access create account page . Which is wrong customer need to redirect on home page or my account section.

**Expected behavior**
Customer need to redirect on home page or my account section.

**Please let us know what packages this bug is in regards to:**
- [x] `venia-concept`

### PR DESCRIPTION
**Pull request by gauravagarwal1001**
Issue #1549 Fixed (Sign in customer access create account page)
## Description

1. Sign in with customer detail .
2. Now open create account page 
3. You are able to access create account page . Which is wrong customer need to redirect on home page or my account section.

## Verification Steps

1. Sign in with customer detail .
2. Now open create account page 
3. Now customer redirect to homepage.

## Proposed Labels for Change Type/Package

- [ ] major (e.g x.0.0 - a breaking change)
- [ ] minor (e.g 0.x.0 - a backwards compatible addition)
- [x] patch (e.g 0.0.x - a bug fix)

Closes #1549

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/pwa-studio?pullRequest=1548) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/pwa-studio?pullRequest=1548) before we can accept your contribution.<br/>

**Comment by gauravagarwal1001** [comment]
> 	Fails
> no_entry_sign 	
> 
> No linked issue found. Please link a relevant open issue by adding the text "closes #<issue_number>" in your PR.
> no_entry_sign 	A version label is required. A maintainer must add one.
> 	Messages
> book 	We are currently working on automating the PR metadata checks. Until that time, you may see failures related to labels/description/linked issues/etc even if you have fixed the problem. Failures will persist until the next push (assuming they are fixed).
> 
> Generated by no_entry_sign [dangerJS](https://danger.systems/js) against [4c4ca2a](https://github.com/magento/pwa-studio/commit/4c4ca2a08eed03cb2c2a68455899db571f161abe)

Issue linked #1549

**Comment by supernova-at** [comment]
Hi @gauravagarwal1001 , I am having some trouble reproducing this issue.

> 1. Sign in with customer detail .
> 2. Now open create account page

From https://magento-venia-develop.now.sh/ (the latest `develop` code) can you provide a screenshot that shows how to access the create account page after I am signed in?

Once I am signed in I no longer can access the create account page.

Thanks!

**Comment by gauravagarwal1001** [comment]
> Hi @gauravagarwal1001 , I am having some trouble reproducing this issue.
> 
> > 1. Sign in with customer detail .
> > 2. Now open create account page
> 
> From https://magento-venia-develop.now.sh/ (the latest `develop` code) can you provide a screenshot that shows how to access the create account page after I am signed in?
> 
> Once I am signed in I no longer can access the create account page.
> 
> Thanks!

After signed in you can directly access create account page with this link 
https://magento-venia-develop.now.sh/create-account

Below is the screenshot .. Signed in customer access create account page.
error

File: packages/venia-concept/src/components/CreateAccount/createAccount.js [hunk 1]
```diff
@@ -74,7 +74,10 @@ class CreateAccount extends Component {
 
     render() {
         const { errorMessage, handleSubmit, initialValues, props } = this;
-        const { classes } = props;
+        const { classes, isSignedIn } = props;
+        if (isSignedIn) {
+            return <Redirect to="/" />;
```
Comment (supernova-at) [inline packages/venia-concept/src/components/CreateAccount/createAccount.js]: This throws a `ReferenceError: Redirect is not defined` error.

Be sure to include `Redirect` in the `import` statements at the top of the file 👍

File: packages/venia-concept/src/components/CreateAccount/createAccount.js [hunk 2]
```diff
@@ -74,7 +74,10 @@ class CreateAccount extends Component {
 
     render() {
         const { errorMessage, handleSubmit, initialValues, props } = this;
-        const { classes } = props;
+        const { classes, isSignedIn } = props;
+        if (isSignedIn) {
```
Comment (supernova-at) [inline packages/venia-concept/src/components/CreateAccount/createAccount.js]: I wouldn't say no to a blank line between the destructuring and the conditional here, either 😄 , will help with readability.

**Comment by supernova-at** [comment]
Hi @gauravagarwal1001, there were some venia refactors that just got merged and so this PR has some conflicting files. Please let us know if you're unsure how to resolve the conflicts.

**Comment by gauravagarwal1001** [comment]
> Hi @gauravagarwal1001, there were some venia refactors that just got merged and so this PR has some conflicting files. Please let us know if you're unsure how to resolve the conflicts.

Yes I don't know what's the conflicts in this files.. How to check conflicts of these files ?

**Comment by sirugh** [comment]
@gauravagarwal1001 Most everything was moved to `venia-ui/`. You should be able to move the two files from `venia-concept/src/*` to `venia-ui/lib/*`.

**Comment by gauravagarwal1001** [comment]
> @gauravagarwal1001 Most everything was moved to `venia-ui/`. You should be able to move the two files from `venia-concept/src/*` to `venia-ui/lib/*`.

I created new pr #1559

**Pull request closed without merging**